File: scripts/check_syl.js
const fs = require('fs');
const content = fs.readFileSync('src/lib/data/ncertSyllabus.ts', 'utf-8');
const match = content.match(/export const NCERT_SYLLABUS[\s\S]*?=\s*({[\s\S]*});/);
if (match) {
    const sylString = match[1];
    // Need a safer way to parse TS object to JSON, but we can just regex the chapters
}

File: scripts/check_syl.py
import json, re

with open('src/lib/data/ncertSyllabus.ts', 'r') as f:
    content = f.read()

# very rough extraction
match = re.search(r'export const NCERT_SYLLABUS.*=\s*({.*});', content, re.DOTALL)
if match:
    syl = match.group(1)
    # This is TS, not strict JSON (keys might not be quoted, etc)
    # Let's just parse it via node
    pass

